pkg/podres/mock_PodresourceClient.go

```diff
@@ -0,0 +1,76 @@
+package podres
```

Comment:
is the capital P on purpose?

Reply:
Conforming to the naming conversions used by NFD (e.g. https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/pkg/labeler/mock_LabelerClient.go)